lib/museum.rb

```diff
@@ -0,0 +1,17 @@
+class Museum
+
+  attr_reader :name, :exhibits
+  def initialize(name)
+    @name = name
+    @exhibits = []
+  end
+
+  def add_exhibit(exhibit_name)
+    @exhibits << exhibit_name
+  end
+
+  def recommend_exhibits(patron)
+    @exhibits.find_all {|exhibit|
+      exhibit.include?(patron.interests)}
```

Comment:
The issue here is we aren't checking the exhibit name we are looking at the entire exhibit object and we want to check if the patron's interested include the exhibit name. 
```suggestion
      patron.interests.include?(exhibit.name)}
```